src/main/java/frc/robot/Constants.java

```diff
@@ -86,6 +89,15 @@ public static class Drive {
 
     /** The number of seconds that we want to taxi for */
     public static double TAXI_AUTO_DURATION_SECONDS = 4.0;
+
+    /** The amount of time we want/need to drive away from the amp in auto. */
+    public static double DRIVE_AWAY_FROM_AMP_TIME = 2.0;
+
+    /** The time we use to back away from the amp in auto. */
+    public static double BACK_UP_FROM_AMP_TIME = 0.5;
+
+    /** Number of radians per second that we want to turn while seeking an unseen target. */
+    public static double LIMELIGHT_SEEKING_RADIANS_PER_SECOND = Math.PI / 2;
```

Comment:
Move these into a subclass for the auto

Reply:
Since this will possibly be used outside of just auto, we moved it to the base `Constants.Limelight` class, since it shouldn't have to change from one Limelight to another.